packages/inter-protocol/src/proposals/add-auction.js

```diff
@@ -79,6 +81,11 @@ export const addAuction = async (
     auctioneerInstallationP,
   ]);
 
+  // save the auctioneer instance so we can manage it later
+  const retiredContractInstances = await retiredContractInstancesP;
+  const legacyInstance = legacyKit.instance;
+  retiredContractInstances.init('auction-vat157', legacyInstance);
```

Comment:
maybe add a note that this suffix needs to be changed each time this proposal is re-used

Reply:
done